cpp/CMakeLists.txt

```diff
@@ -139,12 +140,12 @@ function(bbp_enable_cmake_formatting)
     if(EXISTS ${PROJECT_SOURCE_DIR}/.cmake-format.changes.yaml)
       execute_process(
         COMMAND
-          ${PYTHON_EXECUTABLE} "${CMAKE_CURRENT_SOURCE_DIR}/cmake/cpplib.py" merge-yaml
```

Comment:
In fact, both commands are `merge-yaml` but if those are clang-tidy yaml, so we do something special, I guess that an option instead of a new command, is more clear.